devenv/loadtest/run.sh

```diff
@@ -1,17 +1,21 @@
 #/bin/bash
 
-PWD=$(pwd)
+cd "$(dirname $0)"
 
 run() {
-  duration='15m'
-  url='http://localhost:3000'
-  vus='2'
+  local duration='15m'
+  local url='http://localhost:3000'
+  local vus='2'
+  local iterationsOption=''
 
-  while getopts ":d:u:v:" o; do
+  while getopts ":d:i:u:v:" o; do
     case "${o}" in
 				d)
             duration=${OPTARG}
             ;;
+        i)
+            iterationsOption="--iterations ${OPTARG}"
```

Comment:
Could be nice to add it to the doc here: https://github.com/grafana/grafana-image-renderer/blob/master/devenv/loadtest/README.md with some options that you find useful for example :)